.github/workflows/fuzz_build.yml

```diff
@@ -0,0 +1,32 @@
+name: fuzz-build
+
+on:
+  pull_request:
+    paths:
+      - "fuzz/**"
+  push:
+    branches: master
+
+defaults:
+  run:
+    working-directory: fuzz
+
+env:
+  CARGO_INCREMENTAL: 0
+  RUSTFLAGS: "-Dwarnings"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: RustCrypto/actions/cargo-cache@master
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+      - name: Install cargo-fuzz
+        run: |
+          cargo install cargo-fuzz
```

Comment:
It's better to install a pre-compiled version, e.g. see how we [install `cross`](https://github.com/RustCrypto/actions/blob/master/cross-install/action.yml#L10-L13).

Also, maybe we don't need to install cargo-fuzz in the first place and just use `cargo build`?